README.md

```diff
@@ -123,6 +123,26 @@ let x = 0.0
 end
 ```
 
+Note that the let block is used here to create a scope to allow assignment of 
```

Comment:
That code won't work without the `let`block or another block that creates a local scope. At the REPL, you need the `let` block or a `global` keyword for `x`. Inside a function, it's okay without the `let` block.